src/world.rs

```diff
@@ -305,6 +305,17 @@ impl World {
         })
     }
 
+    /// Given the ID of an entity without a generation, try to find the matching generation.
+    ///
+    /// This function is useful if you know there's an element with a given index, but you
+    /// don't know its generation.
+    ///
+    /// # Safety
+    /// `id` must correspond to a currently live `Entity`. A despawned or never-allocated `id` will produce undefined behavior.
+    pub unsafe fn resolve_unknown_gen(&self, id: u32) -> Entity {
```

Comment:
```suggestion
    pub unsafe fn entity_from_id(&self, id: u32) -> Entity {
```
On further consideration, how do you feel about renaming this? We don't currently expose any user-visible notion of `generation`, so I think this is a bit more obvious (not that we want to encourage incautious use...).

Reply:
That makes sense to me. I was struggling with finding a good name and this seems to fit, so if it satisfies you I'm down with it.